docs/user-guide/external-articles.md

```diff
@@ -4,3 +4,4 @@
 - [Accessing Kubernetes Pods from Outside of the Cluster](http://alesnosek.com/blog/2017/02/14/accessing-kubernetes-pods-from-outside-of-the-cluster)
 - [Kubernetes - Redirect HTTP to HTTPS with ELB and the Ingress-Nginx Controller](https://dev.to/tomhoule/kubernetes---redirect-http-to-https-with-elb-and-the-nginx-ingress-controller)
 - [Configure Nginx Ingress Controller for TLS termination on Kubernetes on Azure](https://blogs.technet.microsoft.com/livedevopsinjapan/2017/02/28/configure-nginx-ingress-controller-for-tls-termination-on-kubernetes-on-azure-2/)
+- [Secure your Nginx Ingress controller behind Cloud Armor or Identity-Aware Proxy (IAP)](https://medium.com/google-cloud/secure-your-nginx-ingress-controller-behind-cloud-armor-805d6109af86)
```

Comment:
The article is behind a paywall :(

Reply:
Oh you are right, good catch, just tried in Incognito mode:
<img width="365" alt="image" src="https://github.com/kubernetes/ingress-nginx/assets/11720844/e75af0d6-cbf7-4db4-9645-10b7c5fb4dda">

I just updated the URL to be publicly available, for anyone, even non-Medium member.